components/builder-web/app/shared/statuspage/statuspage-indicator.component.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2016-2017 Chef Software Inc. and/or applicable contributors
```

Comment:
I recently learned these notices should now just be `2018`. (Time of creation, I guess.)